content/GettingStarted/SCALEReleaseNotes.md

```diff
@@ -57,7 +57,7 @@ More details are available from [23.10 Upgrades]({{< relref "23.10Upgrades.md" >
 {{< truetable >}}
```

Comment:
I think with the rework of the Software Releases page we might be ok to remove this now and instead point up to that URL?